Formula/step.rb

```diff
@@ -25,6 +18,7 @@ def install
     (buildpath/"src/github.com/smallstep/cli").install contents
 
     cd "src/github.com/smallstep/cli" do
+      ENV["VERSION"] = "v0.8.5"
```

Comment:
Unless you are dependent on the v you can use `version` instead of using this that needs to be updated with every version.

Reply:
Did some debugging and realized the version wasn't being picked up by `make build` b/c `Dir["*"]` does not include dot files (.git, etc.). Think I've got it working w/out needing to set the `VERSION`.